src/main/java/org/spongepowered/common/mixin/api/mcp/entity/passive/WolfEntityMixin_API.java

```diff
@@ -45,4 +45,14 @@
         manipulators.add(this.get(SittingData.class).get());
         manipulators.add(new SpongeAggressiveData(this.shadow$isAngry()));
     }
+
+    @Override
+    protected Set<Value.Immutable<?>> api$getVanillaValues() {
+        Set<Value.Immutable<?>> vanillaValues = super.api$getVanillaValues();
+
+        vanillaValues.add(this.angry().asImmutable());
```

Comment:
Do we have the `sitting` key value for wolves? IF it's just a key, you can:
```suggestion
        vanillaValues.add(this.angry().asImmutable());
        this.getValue(Keys.WOLF_SITTING).map(Value::asImmutable).ifPresent(vanillaValues::add);
```